creator-node/test/StateMonitoringQueue.test.js

```diff
@@ -95,6 +101,84 @@ describe('test StateMonitoringQueue initialization and logging', function () {
       .be.fulfilled.and.be.empty
   })
 
+  it('processes jobs with expected data and returns the expected results', async function () {
+    // Mock StateMonitoringQueue to have processStateMonitoringJob return dummy data
+    const expectedResult = { jobFailed: false, test: 'test' }
+    const processStateMonitoringJobStub = sandbox
+      .stub()
+      .resolves(expectedResult)
+    const MockStateMonitoringQueue = proxyquire(
+      '../src/services/stateMachineManager/stateMonitoring/StateMonitoringQueue.js',
+      {
+        './processStateMonitoringJob': processStateMonitoringJobStub
+      }
+    )
+
+    // Verify that StateMonitoringQueue returns our dummy data
+    const job = {
+      id: 9,
+      data: {
+        lastProcessedUserId: 2,
+        discoveryNodeEndpoint: 'http://test_endpoint.co',
+        moduloBase: 1,
+        currentModuloSlice: 2
+      }
+    }
+    await expect(
+      new MockStateMonitoringQueue().processJob(job)
+    ).to.eventually.be.fulfilled.and.deep.equal(expectedResult)
+    expect(processStateMonitoringJobStub).to.have.been.calledOnceWithExactly(
+      job.id,
+      job.data.lastProcessedUserId,
+      job.data.discoveryNodeEndpoint,
+      job.data.moduloBase,
+      job.data.currentModuloSlice
+    )
+  })
+
+  it('returns default result when processing a job fails or logging fails', async function () {
+    // Mock StateMonitoringQueue to have processStateMonitoringJob return dummy data
```

Comment:
whoops, this should say it's throwing not returning dummy data. will fix in a followup PR